README.md

```diff
@@ -54,5 +54,7 @@ The course will also be taught [interally](internal_workshop.md).
     * The command for Windows/Anaconda users will be:
      `conda env create -f environment.yml`
 
+    You can find additional instructions in our [setup](setup.md)
```

Comment:
I would de-indent the Environment paragraph, because now I see that it is rendered as a code block even if only a couple of lines are formatted as code. Maybe it's the indentation to cause the problem.